src/services/microservice/transport-queue.service.ts

```diff
@@ -0,0 +1,74 @@
+import { randomUUID } from 'crypto';
+import { clearInterval } from 'timers';
+
+import { CompleteTransportQueueItem, ExternalTransportType, TransportQueueItem } from '../../models';
+import { NATIVE_SERVICES } from '../../models/constants/native-services';
+import { Injector } from '../../models/dependency-injection/injector.service';
+import { TransportSubjects } from '../subjects/subjects';
+
+export class TransportQueueService {
+  private enqueueInterval: any;
+  private queues: Record<ExternalTransportType, Set<CompleteTransportQueueItem>> = {
+    mqtt: new Set<CompleteTransportQueueItem>(),
+    redis: new Set<CompleteTransportQueueItem>(),
+  };
+
+  private enqueueItems(): void {
+    for (const key of Object.keys(this.queues)) {
+      [...this.queues[key]].forEach((item) => TransportSubjects.sendTransportSubject.next(item));
+    }
```

Comment:
Could use `Object.values`
```suggestion
    for (const queues of Object.values(this.queues)) {
      queues.forEach((item) => TransportSubjects.sendTransportSubject.next(item));
    }
```

Reply:
Yep, I will implement this change.